e2e/unsubscribed.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/unsubscribed');
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe('The unsubscribe page', () => {
+  test('The page renders with correct title', async () => {
+    await expect(page).toHaveTitle(
+      'You have been unsubscribed | freeCodeCamp.org'
+    );
+  });
+
+  test('The page renders successful unsubscription message', async () => {
+    const heading = page.getByTestId('unsubscribed-heading');
+    const description = page.getByTestId('unsubscribed-description');
+
+    await expect(heading).toContainText(translations.misc.unsubscribed);
+    await expect(description).toContainText(translations.misc['keep-coding']);
+  });
+
+  test('The page renders resubscribe button when user access /unsubscribed/:unsubscribeId route', async () => {
+    const unsubscribeId = 'foo@baz.com';
```

Comment:
Probably need to use a real id from the mock user? @ojeytonwilliams what do you reckon?

Reply:
I was thinking about it, but the href of resubscribe link is actually based on the url. 

If I go to www.freecodecamp.org/unsubscribed/foo@bar.com the href will be https://api.freecodecamp.org/resubscribe/foo@bar.com

![CleanShot 2023-10-02 at 12 29 35](https://github.com/freeCodeCamp/freeCodeCamp/assets/58401630/bd1fbae1-438f-4b62-aec0-7e9b8a797243)

Reply:
Sorry - I did not clarify this earlier. What I meant was the `unsubscribeId` is not an email address. It is a unique string (uuid-style) unique to a user. We need to use that id for whatever it is for the test/development user. 

I'll check this for you.

Reply:
Hey @raisedadead, I'm just thinking that maybe it doesn't have to be ID for test/development user, because `unsubscribed/:unsubscribedId` can be accessed even when you're not logged in. What about using just some mock uuid?


```suggestion
    const unsubscribeId = 'b34de091-f627-4649-9fd2-44887fe7731b';
```